README.md

```diff
@@ -273,6 +289,47 @@ class GoogleMapsType extends AbstractType
 }
 ```
 
+For PHP 8.1 and newer, you can use the `#[AsUiElement]` attribute to define your UiElement. For example:
+
+```php
+<?php
+
+// ...
+
+use MonsieurBiz\SyliusRichEditorPlugin\Attribute\AsUiElement;
+
+#[AsUiElement(
+    code: 'app.google_maps',
+    icon: 'map pin',
+)]
+class GoogleMapsType extends AbstractType
+// ...
+```
+
+The title, description and templates values are generated automatically from the code. In this example :
+
+- the title will be `app.ui_element.google_maps.title`,
+- the description will be `app.ui_element.google_maps.description`,
+- the admin template will be `/Admin/UiElement/google_maps.html.twig`, 
+- and the front template will be `/Shop/UiElement/google_maps.html.twig`.
+
+But you can override them if you want:
+
+```php
+#[AsUiElement(
+    code: 'app.google_maps',
+    title: 'my_cusom.title', // Use your own translation key or a string
+    description: 'my_custom.description',
+    icon: 'map pin',
+    templates: new TemplatesUiElement(
+        adminRender: 'MyCusomPath/google_maps.html.twig',
+        frontRender: 'MyCusomPath/google_maps.html.twig',
+    ),
+    uiElement: GoogleMapsUiElement::class, // Use your own UiElement class
+    tags: ['map'], // Add some tags to filter the UiElement
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Maybe we can add example for the wireframe ?

Reply:
@maximehuran it's done https://github.com/monsieurbiz/SyliusRichEditorPlugin/compare/bef7b6614beccfcb215b6d4c5c386bbc03078a2a..4c8f81a1088db3eece7c7d79fc9d4ade2aa1fbe5